src/Lean/Parser/Term.lean

```diff
@@ -51,6 +51,7 @@ namespace Term
 /- Built-in parsers -/
 
 @[builtinTermParser] def byTactic := leading_parser:leadPrec ppAllowUngrouped >> "by " >> Tactic.tacticSeq
+def byTactic' := leading_parser:leadPrec ppAllowUngrouped >> "by " >> Tactic.tacticSeq
```

Comment:
```suggestion
def byTactic' := leading_parser "by " >> Tactic.tacticSeq
```

The `ppAllowUngrouped` only has an effect for category parsers.  I also don't think the precedences do anything for non-category parsers.